### ISSUE TITLE
Can't compile on Windows 10, VS 2019 Community

### ISSUE THREAD

**Issue by caskater4** [veliovgroup/request-extra#4]
I'm unable to build from source on Windows 10 with VS 2019 Community installed. I receive the following errors.

```
nasm.exe' is not recognized as an internal or external command,
  operable program or batch file.
D:\gitlab\launcher\node_modules\node-libcurl\build\curl-for-windows\openssl\openssl.targets(28,5): error MSB3721: The command "call nasm.exe "-f win64" "-DNEAR" "-Ox" "-g" "-o" "Release\obj\openssl\aesni-mb-x86_64.obj" "..\..\..\curl-for-windows\openssl\config\archs\VC-WIN64A\asm\crypto\aes\aesni-mb-x86_64.asm"" exited with code 1. [D:\gitlab\launcher\node_modules\node-libcurl\build\curl-for-windows\openssl\openssl.vcxproj]

gyp ERR! build error
gyp ERR! stack Error: `C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Current\Bin\MSBuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (D:\gitlab\launcher\node_modules\node-gyp\lib\build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:376:20)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:277:12)
gyp ERR! System Windows_NT 10.0.19042
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "D:\\gitlab\\launcher\\node_modules\\node-gyp\\bin\\node-gyp.js" "build" "--fallback-to-build" "--module=D:\\gitlab\\launcher\\node_modules\\node-libcurl\\lib\\binding\\node_libcurl.node" "--module_name=node_libcurl" "--module_path=D:\\gitlab\\launcher\\node_modules\\node-libcurl\\lib\\binding" "--napi_version=8" "--node_abi_napi=napi" "--napi_build_version=0" "--node_napi_label=node-v83"
gyp ERR! cwd D:\gitlab\launcher\node_modules\node-libcurl
gyp ERR! node -v v14.17.0
gyp ERR! node-gyp -v v7.1.0
gyp ERR! not ok
node-pre-gyp ERR! build error
node-pre-gyp ERR! stack Error: Failed to execute 'C:\Program Files\nodejs\node.exe D:\gitlab\launcher\node_modules\node-gyp\bin\node-gyp.js build --fallback-to-build --module=D:\gitlab\launcher\node_modules\node-libcurl\lib\binding\node_libcurl.node --module_name=node_libcurl --module_path=D:\gitlab\launcher\node_modules\node-libcurl\lib\binding --napi_version=8 --node_abi_napi=napi --napi_build_version=0 --node_napi_label=node-v83' (1)
node-pre-gyp ERR! stack     at ChildProcess.<anonymous> (D:\gitlab\launcher\node_modules\node-pre-gyp\lib\util\compile.js:83:29)
node-pre-gyp ERR! stack     at ChildProcess.emit (events.js:376:20)
node-pre-gyp ERR! stack     at maybeClose (internal/child_process.js:1055:16)
node-pre-gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:288:5)
node-pre-gyp ERR! System Windows_NT 10.0.19042
node-pre-gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "D:\\gitlab\\launcher\\node_modules\\node-pre-gyp\\bin\\node-pre-gyp" "install" "--fallback-to-build"
node-pre-gyp ERR! cwd D:\gitlab\launcher\node_modules\node-libcurl
node-pre-gyp ERR! node -v v14.17.0
node-pre-gyp ERR! node-pre-gyp -v v0.15.0
node-pre-gyp ERR! not ok
```

**Comment by dr-dimitru** [comment]
Hello @caskater4 ,

Sorry for the late reply, have you figured it out?
I believe complier missing one f the dependencies "nasm.exe"

Feel free to close it in case if the issue is solved on your end.

**Comment by dr-dimitru** [comment]
@caskater4 build issue should be solved in the [v2.3.3](https://github.com/veliovgroup/request-extra/releases/tag/2.3.3).

Feel free to reopen it in case if the issue is still persists on your end.
